### PR TITLE
Bump max-version to 11.0.0.0

### PR DESCRIPTION
**Pull request by phil-davis**
1) Bump max-version to 11.0.0.0 like we did with all other apps
2) Copy changes from https://github.com/sabre-io/dav/blob/master/tests/Sabre/DAV/Auth/Backend/AbstractBearerTest.php to make unit tests pass against core master
3) The last 2 drone matrix entries were identical, testing PHP 7.2 with core master - change one of them so it tests with PHP 7.1

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -174,7 +174,7 @@ matrix:
       TEST_SUITE: phpunit
       DB_TYPE: sqlite
 
-    - PHP_VERSION: 7.2
```
Comment (DeepDiver1975) [inline .drone.yml]: why was this necessary?

File: tests/Unit/Sabre/AbstractBearerTest.php [hunk 2]
```diff
@@ -68,27 +65,23 @@ function testRequireAuth() {
...
 		);
-
 	}
-
 }
 
-
-class AbstractBearerMock extends AbstractBearer {
-
+class AbstractBearerMock extends AbstractBearer
```
Comment (DeepDiver1975) [inline tests/Unit/Sabre/AbstractBearerTest.php]: code style :boom:

File: .drone.yml (hunk 1 above)
Comment (phil-davis) [reply .drone.yml]: Because the matrix entry below already has PHP 7.2 and was identical.

File: tests/Unit/Sabre/AbstractBearerTest.php (hunk 2 above)
Comment (phil-davis) [reply tests/Unit/Sabre/AbstractBearerTest.php]: :( I pasted from the Sabre Dav code.

File: .drone.yml (hunk 1 above)
Comment (DeepDiver1975) [reply .drone.yml]: Cool - THX

File: tests/Unit/Sabre/AbstractBearerTest.php (hunk 2 above)
Comment (DeepDiver1975) [reply tests/Unit/Sabre/AbstractBearerTest.php]: it's okay - we add php-csfixer in a followup pr
